### PR TITLE
Add a script to auto-generate the normal/visual mode character lookup table

### PR DESCRIPTION
**Pull request by yegappan**

### CODE REVIEW

**Comment by brammool** [comment]
Thanks, it actually looks simpler than I anticipated.
I expect it won't work for EBCDIC, but I started a discussion to remove EBCDIC support.

Can you write the script in Vim9 syntax?  Using a :def function checks for errors and will run faster.

File: src/Make_mvc.mak [hunk 1]
```diff
@@ -1446,6 +1446,13 @@ clean: testclean
 cmdidxs: ex_cmds.h
 	vim --clean -X --not-a-term -u create_cmdidxs.vim
 
+# Run vim script to generate the normal/visual mode command lookup table.
+# This only needs to be run when a new normal/visual mode command has been
+# added.  If this fails because you don't have Vim yet, first build and
+# install Vim without changes.
+nvcmdidxs: nv_cmdidxs.h
+	vim --clean -X --not-a-term -u create_nvcmdidxs.vim
```
Comment (ichizok) [inline src/Make_mvc.mak]: This `vim` must be what is built from the "current" sources, unlike of `make cmdidxs`, mustn't it?
Namely, I feel, needs the these steps when update `nv_cmdidxs.h`:

1. change `nv_cmds[]` in normal.c
2. build vim
3. do `make nvcmdidxs` using vim built in step 2
4. rebuild vim (if need)

Comment (yegappan) [reply src/Make_mvc.mak]: Yes. I will update the comments to reflect that.

Comment (ichizok) [reply src/Make_mvc.mak]: Should use `./$(VIMTARGET)` instead of `vim`.

Comment (yegappan) [reply src/Make_mvc.mak]: Thanks for the suggestion. I have updated the PR.

**Pull request closed without merging**